genienlp/predict.py

```diff
@@ -164,7 +164,13 @@ def parse_argv(parser):
         default=[0],
         help='ngrams of this size cannot be repeated in the output. 0 disables it.',
     )
-    parser.add_argument('--max_output_length', default=150, type=int, help='maximum output length for generation')
+    parser.add_argument('--max_output_length', type=int, help='maximum output length for generation')
```

Comment:
Removing the default value for `max_output_length` can -under rare circumstances- result in some paraphrasing models requiring a smaller prediction batch size. So please revert this change.

Reply:
min/max_output_length should match with what was provided for training right? (If we provide defaults here they will override train args.)